docs/README-migration.md

```diff
@@ -54,7 +54,7 @@ SDL_PauseAudioDevice() is only used to pause audio playback. Use SDL_PlayAudioDe
 
 SDL_FreeWAV has been removed and calls can be replaced with SDL_free.
 
-SDL_AudioCVT interface is removed, SDL_AudioStream can be used instead.
+SDL_AudioCVT interface is removed, SDL_AudioStream can be used instead.  SDL_ConvertSamples() has been added as a helper function.
 
 Code that used to look like this:
 ```c
```

Comment:
You can use the new function in the sample code below.

Reply:
updated